src/components/L2/Headers/L2BackHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Row, Icon } from '@tlon/indigo-react';
+
+import useRoller from 'lib/useRoller';
+import { useHistory } from 'store/history';
+import { useRollerStore } from 'store/roller';
+
+import './L2BackHeader.scss';
+import { useNetwork } from 'store/network';
+import { useWallet } from 'store/wallet';
+import { toBN } from 'web3-utils';
+import BN from 'bn.js';
+
+export interface L2BackHeaderProps {
+  back?: () => void;
+  hideBalance?: boolean;
+}
+
+const L2BackHeader = ({ back, hideBalance = false }: L2BackHeaderProps) => {
+  const { config } = useRoller();
+  const { nextRoll, currentL2 } = useRollerStore(store => store);
+  const { pop }: any = useHistory();
+  const { wallet }: any = useWallet();
+  const { web3 }: any = useNetwork();
+
+  const [ethBalance, setEthBalance] = useState<BN>(toBN(0));
+
+  const goBack = useCallback(() => {
+    if (back) {
+      back();
+    } else {
+      pop();
+    }
+  }, [back, pop]);
+
+  useEffect(() => {
+    console.log('loaded config in L2BackHeader:', config);
```

Comment:
Let's either log this only in development / tmp environment, or remove completely